### ISSUE TITLE
7.7RC2 Save button on Project Templates does not work

### ISSUE THREAD

**Issue by 2xaronl** [salesagility/SuiteCRM#1827]
In 7.7RC2 I'm getting
```
ReferenceError: SugarWidgetScheduler is not defined  index.php:932:3
ReferenceError: SugarWidgetScheduler is not defined  index.php:927:5
```
in my browser console when I click the Save button in the Project Template full entry form.  The project template is not saved.

Expected Behavior
The record should save.

Actual Behavior
Nothing except the error above.

Steps to Reproduce
Attempt to save a new Project Template from the Create Project Template form

Context
New project templates cannot be created.

Your Environment
SuiteCRM Version used: 7.7RC2
Browser name and version: Firefox 47.0.1, Chrome 51.0.2704.103 m
Environment name and version: MySQL 5.6, PHP 5.5.9
Operating System and version: Ubuntu 14.04.4 LTS

**Comment by c0cl3c** [comment]
I am getting a similar issue. Hitting Save nothing appears to be happening on the screen.

Installed on Ubuntu 16.04 with PHP 7.0. Accessing via Chrome 51.0.2704 on OSX.

**Comment by 2xaronl** [comment]
After upgrading from 7.7RC2 to 7.7 production release I revisited this issue and was able to save a project template but I have been unable to add users as resources.  After selecting a user from the pop-up dialog the project template page refreshes but no user is added.  I see no error messages.

**Comment by samus-aran** [comment]
Hi @2xaronl Can you raise that as a new issue please if still exists in 7.7.2

for original issue this should be resolved in v7.7.2
https://github.com/salesagility/SuiteCRM/releases/tag/v7.7.2
